### DISCUSSION TITLE
Introducing Myself and What's Next?

### DISCUSSION THREAD

**Discussion by arimgibson** [arimgibson/Deadname-Remover#511] [General]
Hi all! My name's Ari (they/them), and as Willow was no longer able to devote her time to this project, I've been chosen to take over the maintenance. I wanted to start by just collecting some community feedback about what's most important for the future of the project so that I have something to start running off of!

I also wanted to put out a call that I'd love some help triaging and responding to the submitted issues with this plugin. There's a good bit of work to be done weeding through, removing duplicates, adding milestones/Kanban cards for fixes and features, etc. If anyone's interested in assisting with this process to clean the repository up a little bit, I'd appreciate it. Feel free to respond here and we can coordinate a bit.

Thanks! I know these discussion posts tend to not be very active, but I'm hoping to gather at least a touch of community input.

cc @Gusted as I'd love to know if you're still interested in being a collaborator on this project and have obviously been working on it much longer than me 😛

**Comment by bringmecheese** [comment]
Hey Ari, the extension is great and I'm so happy to hear that it's still being taken care of today. Thank you all for your efforts!

A feature I'd love to see if possible would be an option to have some sort of indicator/highlighted text if the deadname was replaced? (I was thinking of scenarios where I might accidentally call someone with my deadname the wrong name aka my own, haha and/or whenever I'd like to update someone about my name.)

**Reply by arimgibson** [reply]
Thanks for the input Lance! This feature is actually ready to implemented thanks to #199, but I have yet to integrate it into the extension. Expect it to be merged in with the release of Version 2.0,0, which is what I'm putting my energy towards right now.

The task of integrating that feature can also be found under the project [⏩ Maintenance Catch-Up After Migration](https://github.com/arimgibson/Deadname-Remover/projects/1) . I'll make sure to add that "card"/task (not sure if you're familiar with Kanban) to the [Version 2.0.0 Milestone](https://github.com/arimgibson/Deadname-Remover/projects/3). Lastly, I'll give you a ping when that feature is released! And feel free to give any feedback on #199 as well by adding a comment in there.

**Comment by junedabat** [comment]
Hello there! I've noticed the extension hasn't been updated on [Firefox Addons](https://addons.mozilla.org/en-US/firefox/addon/deadname-remover/) since 2 years ago, do you know when it will be updated next?
